src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -19,14 +21,12 @@ if NOT [%ERRORLEVEL%]==[0] (
     goto :done
   )
 
-
 REM Config Certs
 REM we need the direcotry to save the test.crl file. We are investigate a way to get rid of it
 REM
 md c:\wcftest
-call %~dp0..\..\..\..\bin\Wcf\tools\CertificateGenerator\CertificateGenerator.exe %*
+%~dp0..\..\..\..\bin\Wcf\tools\CertificateGenerator\CertificateGenerator.exe
```

Comment:
It's removed by accident. I'll add the call back. The call to an exe does not make any difference, but it makes difference for cmd scripts.